### DISCUSSION TITLE
Linux

### DISCUSSION THREAD

**Discussion by fnfgo** [3kh0/ext-remover#545] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there anyway to force install linux on school chromebook? If theres not full permission to call me a dumbass

**Comment by obkduishinijf** [comment, marked as answer]
with or without using sh1mmer?

**Reply by fnfgo** [reply]
Without

**Reply by obkduishinijf** [reply]
> Without

probably not, you could try chrome://crostini-installer but there's a very low chance it'll work.

**Reply by fnfgo** [reply]
Okay thank you
